snuba/datasets/pipeline/query_pipeline.py

```diff
@@ -0,0 +1,42 @@
+from abc import ABC, abstractmethod
```

Comment:
The pipeline logic is likely going to grow a lot. I would take it out of the dataset module and make it a top level module.

Reply:
Moved